tests/test_wcs.py

```diff
@@ -408,26 +408,43 @@ def test_ndvi_xarray(self):
                     [255, 255, 115, 255, 139, 255, 255, 255],
                     [255, 255, 255, 255, 255, 255, 255, 255],
                     [255, 255, 255, 255, 255, 255, 255, 255]
-                ]]),
+                ]], dtype=np.uint8),
                 coords={
                     'band': [1],
+                    'x': [-157.5, -112.5, -67.5, -22.5, 22.5, 67.5, 112.5, 157.5],
                     'y': [78.75, 56.25, 33.75, 11.25, -11.25, -33.75, -56.25, -78.75],
-                    'x': [-157.5, -112.5, -67.5, -22.5, 22.5, 67.5, 112.5, 157.5]
+                    'spatial_ref': 0,
```

Comment:
Why is spatial ref zero? In `attrs` we can see that we have 'EPSG:4326'. Is there an example of the library where this is not zero and do you know the implications? Do we need to set this manually somehow?

Reply:
The field is just there as a default of the rioxarray output.

I'm not totally sure about the details, but it seems, that it's set to 0 when there is more information available.
You can also call 'array.spatial_ref' directly with the following output in our case here:
`<xarray.DataArray 'spatial_ref' ()>
array(0)
Coordinates:
    spatial_ref  int64 0
Attributes:
    crs_wkt:                      GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["...
    semi_major_axis:              6378137.0
    semi_minor_axis:              6356752.314245179
    inverse_flattening:           298.257223563
    reference_ellipsoid_name:     WGS 84
    longitude_of_prime_meridian:  0.0
    prime_meridian_name:          Greenwich
    geographic_crs_name:          WGS 84
    grid_mapping_name:            latitude_longitude
    spatial_ref:                  GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["...
    GeoTransform:                 -180.0 45.0 0.0 90.0 0.0 -22.5`